src/components/InfoPanel/_info-panel.scss

```diff
@@ -5,14 +5,27 @@
   margin-right: 1rem;
   padding: 1rem;
 
+  .p-modal {
+    position: fixed;
+    z-index: 1;
+  }
+
   &__pictogram {
     // Keep height and width in sync with the dimensions of
     // the Topology component.
     background: $color-light;
     border: solid 1px $color-mid-x-light;
     height: 300px;
     margin-bottom: 1rem;
+    position: relative;
     width: 300px;
+
+    i {
```

Comment:
Can you target: `p-icon--expand` as te element could change and it shouldn't effect the styling.

Reply:
done